src/store/index.ts

```diff
@@ -70,10 +70,14 @@ export const store = {
             return settings;
         },
 
-        // TODO: Add reset actions for other modules
-        async resetProfileModule({dispatch}: Context) {
-            return await dispatch('profile/reset');
-        }
+        async resetLocalState({commit}: Context) {
+            return await Promise.all([
```

Comment:
Commits are synchronous methods so using Promise.all does nothing here. Also there's no need to return anything from this function.